src/shell-interface.h

```diff
@@ -50,6 +51,10 @@ struct ShellExternalInterface : ModuleInstance::ExternalInterface {
 
    public:
     Memory() {}
+    char* rawbuffer(unsigned offset, unsigned length) {
```

Comment:
I'm pretty sure now that using `uint8_t*` type for bytes in EVMC was good idea.

Reply:
This is `char*` because the underlying type is `char`.